activity/scraper.py

```diff
@@ -0,0 +1,80 @@
+import requests
+import json
+import datetime
+from dateutil import parser, relativedelta
+
+issuesurl = "https://coala.github.io/gh-board/issues.json"
+today = datetime.datetime.today()
+
+def diff_month(d1, d2):
+	return (d1.year - d2.year) * 12 + d1.month - d2.month
+
+def diff_week(d1, d2):
+	monday1 = (d1 - datetime.timedelta(days=d1.weekday()))
+	monday2 = (d2 - datetime.timedelta(days=d2.weekday()))
+	return (monday1 - monday2).days / 7
```

Comment:
Though I am not sure about this. I think this would be rather `// 7`. We need the floor value for the number of weeks right.

Reply:
Monday1-monday2 is guaranteed to be a multiple of 7. Since, it’s a difference in Monday’s of different weeks...

Reply:
Then maybe `return ((d1-d2).days)//7` would be more simple.

Reply:
The purpose behind doing it this way is to make sure weeks are counted started from Monday. So if the data is scraped on a Thursday, the current week is Monday-Thursday, and not previous Friday-Thursday, since it doesn’t make much sense mentally to have weeks not start from Monday, and also, it maintains consistency with other options(for example, year counts a month partially if it hasn’t ended yet)

Reply:
I've left a comment in the code now to be clear(er) about this for future readers.